opennlp-dl/src/test/java/opennlp/dl/namefinder/NameFinderDLEval.java

```diff
@@ -200,10 +200,8 @@ public void invalidModel() {
 
       final File model = new File("invalid.onnx");
       final File vocab = new File("vocab.txt");
-
-      try (final NameFinderDL nameFinderDL = new NameFinderDL(model, vocab, getIds2Labels(),
-          sentenceDetector)) {
-      }
+      //noinspection resource
+      new NameFinderDL(model, vocab, getIds2Labels(), sentenceDetector);
```

Comment:
If it is ok to leave the ort session/env open in this eval test, the try-with-resources removal is ok. Guess the Compiler warning was, that the final variable is unsed?

Reply:
Compiler warning was similar than in issue description, related to close method call, via try...

Reply:
https://issues.apache.org/jira/projects/OPENNLP/issues/OPENNLP-1646?filter=allissues doesn't answer the question why it's a cure to remove try with resources in the test (but only there and not in other tests as well).